pkg/provider/azure_vmss_cache.go

```diff
@@ -294,24 +313,36 @@ func (ss *ScaleSet) deleteCacheForNode(nodeName string) error {
 	return nil
 }
 
-func (ss *ScaleSet) newAvailabilitySetNodesCache() (*azcache.TimedCache, error) {
+func (ss *ScaleSet) newNonVmssUniformNodesCache() (*azcache.TimedCache, error) {
 	getter := func(key string) (interface{}, error) {
-		vmNames := sets.NewString()
+		klog.V(6).Infof("refresh the cache of NonVmssUniformNodesCache")
+		vmssFlexVMNodeNames := sets.NewString()
```

Comment:
`NonVmssUniformNodesCache` means it's not VMSS uniform cache? while current test failures are all on VMSS uniform nodes, so that's the issue?

Reply:
and I found following logs in vmss uniform node pool cluster:
```
I0914 10:15:02.899048       1 azure_vmss_cache.go:388] Node k8s-agentpool-25348871-vmss000000 has joined the cluster since the last VM cache refresh in nonVmssUniformNodesEntry, refreshing the cache
```

The log is strange since one vmss uniform node is not found and then it tries to refresh nonVmssUniformNodesEntry

Reply:
Per discussion, this issue is resolved by using the correct OrchetrationMode value